variants/esp32-ota-1c0000/partitions.csv

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021 Toitware ApS.
+#
+# This library is free software; you can redistribute it and/or
```

Comment:
Not sure this is the best license for this.

Reply:
Good point. I will update all files to BSD0 in a separate PR.